src/librustc_typeck/collect.rs

```diff
@@ -2583,6 +2586,34 @@ fn codegen_fn_attrs(tcx: TyCtxt<'_>, id: DefId) -> CodegenFnAttrs {
     codegen_fn_attrs
 }
 
+/// Checks if the provided DefId implements a method for a trait which has track_caller applied to the trait prototype.
```

Comment:
This shouldn't pass tidy.

Also, regarding the contents themselves, "implements a method" could be expanded to describe the fact that this is method in a trait impl.